WordPressKit/AccountSettingsRemote.swift

```diff
@@ -90,6 +90,26 @@ public class AccountSettingsRemote: ServiceRemoteWordPressComREST {
                                  })
     }
 
+    /// Validate the current user's username
+    ///
+    /// - Parameters:
+    ///   - username: The new username
+    ///   - success: block for success
+    ///   - failure: block for failure
+    public func validateUsername(to username: String, success: @escaping () -> Void, failure: @escaping (Error) -> Void) {
+        let endpoint = "me/username/validate/\(username)"
+        let path = self.path(forEndpoint: endpoint, withVersion: ._1_1)
```

Comment:
Do we need `self.` here?

Reply:
It's to avoid the `Variable used within its own initial value` error. I left it because it's used in all the other methods of the same service.

Reply:
So it is.  Interesting.  Also kind of annoying that the linter doesn't recognize the full method signature when validating.  `self.` here shouldn't be necessary 😠.  grumble grumble grumble.  I'll stop ranting now.  Thanks for clarifying!